### ISSUE TITLE
Question about `RTSTRUCT` and DICOM images pairing

### ISSUE THREAD

**Issue by ff98li** [Sikerdebaard/dcmrtstruct2nii#46]
Thank you for sharing this amazing package!

I'm using the python API `dcmrtstruct2nii`, which requires the input `rtstruct_file` and `dicom_file`, which means locating the right `dicom_file` for the current `rtstruct_file` is up to the users. I'm new to the DICOM format and I'm currently handling datasets in which there are multiple `RTSTRUCT` and DICOM image directories for a study. So I'm not really sure how to find the right DICOM image directory corresponding to the right `RTSTRUCT`. My current approach is doing the following mapping:
```
For images being referenced by some RTSTRUCT:
(0020,000E) Unique identifier of the series. <----------
-------------------------------------------             | 
For RTSTRUCT referncing CT/MR images:                   |
(3006,0010) Referenced Study Sequence                   |  Check if their series UID match
  (3006,0012) RT Referenced Study Sequence              |
      (3006,0014) RT Referenced Series Sequence         |
          (0020,000E) Unique identifier of the series. <-
```
In other words, I look up the series UID `(0020, 000E)` of a random DICOM in a CT/MR image directory, and then look up the series UID under `(3006, 0010)` in the `RTSTRUCT` DICOM with which I try to find the right DICOM image directory to pair. I'm not sure if this is the right way to find the right `RTSTRUCT` and DICOM image pairing. I will really appreciate it if any user experienced with DICOM could provide more information about handling such cases.

**Comment by Sikerdebaard** [comment]
Hi @ff98li unfortunately I cannot help you with this. This github repo is about converting RTStruct to Nifti and not about DICOM standards. I recommend you ask in a discussion group about DICOM, for example on Reddit r/dicom.
